fastlane/Fastfile

```diff
@@ -99,16 +99,11 @@ platform :ios do
 
     clean_build_artifacts
     commit_version_bump(xcodeproj: "godtools.xcodeproj",
-                        message: "Build number bump for Beta build #{lane_context[SharedValues::BUILD_NUMBER]}",
+                        message: "Build number bump for Beta build #{lane_context[SharedValues::BUILD_NUMBER]} [ci skip]",
```

Comment:
from what I understand the ci skip feature is a part of the github pull request builder jenkins plugin which is what we use for the test jobs when PRs are created. However, we use the git plugin for detecting merged PRs on develop and master. The git plugin does not have this functionality to my knowledge. Here is the documentaion for the yaml you are able to define on each of them so you can check me on that https://docs.openstack.org/infra/jenkins-job-builder/triggers.html#triggers.github-pull-request https://docs.openstack.org/infra/jenkins-job-builder/scm.html#scm.git